js/main.js

```diff
@@ -1,25 +1,118 @@
-const $imgLoad = document.querySelector('#user-photo-url');
-const $image = document.querySelector('#imgshow');
-$imgLoad.addEventListener('input', function (event) {
+const div = document.querySelector('.all-entries');
+const photoURL = document.getElementById('image-url');
+const entriesPage = document.getElementById('entries-page');
 
-  $image.setAttribute('src', event.target.value);
+photoURL.addEventListener('input', function (event) {
+  const img = document.querySelector('img');
```

Comment:
This `querySelector` should be done globally outside this function. By including it here, you're forcing us to traverse the DOM over and over again every time this function gets called.